ca_gazebo/launch/ekuthon_coop_robots.launch

```diff
@@ -0,0 +1,13 @@
+<launch>
```

Comment:
Maintain the naming convention of the rest of the launch files. Call this `create_cooperative_two_rooms` for example.